wallet/src/de/schildbach/wallet/ui/DashPayUserActivityViewModel.kt

```diff
@@ -111,4 +144,67 @@ class DashPayUserActivityViewModel @Inject constructor(
     fun getChainLockBlockHeight(): Int {
         return dashSystemService.system.chainLockHandler.bestChainLockBlockHeight
     }
+
+    private fun observeContactNotifications(dashPayProfile: DashPayProfile) {
+        combine(
+            platformRepo.observeContacts(dashPayProfile.username, UsernameSortOrderBy.DATE_ADDED, true)
+                .distinctUntilChanged(),
+            walletData.observeMostRecentTransaction()
+                .distinctUntilChanged()
```

Comment:
Merging `observeMostRecentTransaction` in here removes the need for `TransactionsLiveData`